web/extensions/core/groupNode.js

```diff
@@ -1188,6 +1188,8 @@ export class GroupNodeHandler {
 		await config.registerType();
 
 		const groupNode = LiteGraph.createNode(`workflow/${name}`);
+		groupNode.title = name;
```

Comment:
This just sets the initial title for the group. We want to use the title rather than the type name so the user can change it later.